### PR TITLE
Add a README which includes an example of how to add a test

### PR DESCRIPTION
**Pull request by jscook2345**
Add a README which includes an example of how to add a test

Attn: @singularity-maintainers

### CODE REVIEW

File: e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+For this example, I'm going to use a topic of `env` or `environment variable tests`.
```
Comment (ArangoGutierrez) [inline e2e/README.md]:
```suggestion
# End to End Tests

This package contains integration tests. 
The tests work with the `SINGULARITY_E2E` environment variable

## Contributing
For this example, we are going to use a topic of `env` or `environment variable tests`.
```

File: e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+* Run your test from your entry-point function using a `go` sub-test.
+```
+//RunE2ETests is the main func to trigger the test suite
+func RunE2ETests(t *testing.T) {
+	err := envconfig.Process("E2E", &testenv)
+        if err != nil {
+        	t.Fatal(err.Error())
+        }
+        
+	t.Log(testenv)
```
Comment (ArangoGutierrez) [inline e2e/README.md]:
```suggestion

```
This was for development and debugging.  I will rm from the current code now that is running

File: e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+	...
+}
+
+//RunE2ETests is the main func to trigger the test suite
+func RunE2ETests(t *testing.T) {
+	err := envconfig.Process("E2E", &testenv)
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+	t.Log(testenv)
```
Comment (ArangoGutierrez) [inline e2e/README.md]:
```suggestion
	
```
same as above

File: e2e/README.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+	t.Run("TestEnv", TestEnv)
+}
+```
+
+* Test your topic using the `e2e` target in the `Makefile`
```
Comment (ArangoGutierrez) [inline e2e/README.md]:
```suggestion
## Running the E2E tests

Test your topic using the `e2e` target in the `Makefile`
```

**Review by ArangoGutierrez — APPROVED** [review]
LGTM

**Comment by jscook2345** [comment]
@ArangoGutierrez 

I think I fixed everything you requested. Please re-review!

**Review by ArangoGutierrez — APPROVED** [review]
lgtm

**Pull request merged**